internal/database/migrations/migrations.go

```diff
@@ -1,13 +1,27 @@
 package migrations
 
 import (
+	"github.com/uwblueprint/shoe-project/config"
 	"github.com/uwblueprint/shoe-project/internal/database/models"
 	"gorm.io/gorm"
 	"syreclabs.com/go/faker"
+	"golang.org/x/crypto/bcrypt"
 )
 
 func CreateTables(db *gorm.DB) error {
-	return db.AutoMigrate(&models.Story{}, &models.Author{})
+	return db.AutoMigrate(&models.Author{}, &models.Story{}, &models.User{})
+}
+
+func CreateSuperUser(db *gorm.DB) error {
+	// Clear current superuser if any
+	db.Where("username = ?", "admin").Delete(models.User{})
+	
+	hashedPassword, _ := bcrypt.GenerateFromPassword([]byte(config.GetSuperUserPassword()), 8);
```

Comment:
I would recommend setting cost for `GenerateFromPassword` to be 10 based on the projects I have worked on and some research I have done in past.